### ISSUE TITLE
Global shorcut

### ISSUE THREAD

**Issue by wiiznokes** [wiiznokes/cosmic-clipboard-manager#20]
2 ways:
- workarround with custom shortcut in cosmic-comp and listen d-bus
- global shortcut from portal, but its not currently impl in cosmic

**Comment by wiiznokes** [comment]
Thinking about this now: how to choose witch instance to popup when there is two panel ?

And two panel on the active workspace ?

**Comment by juanesech** [comment]
Could be possible to show the popup on mouse position? 
I think this could solve the problem on multiple instances and inactive monitor

**Comment by wiiznokes** [comment]
Yeah i thought about it too, it may help, and i do want to support it the future but
- i don't know if there is a wayland protocol for this (it may be possible to clone what cosmic-files-applet do)
- its still hard to know which instance should render the popup

**Comment by wiiznokes** [comment]
I tried to implement the workarround, but applet don't support this feature :(

**Comment by linonetwo** [comment]
CopyQ supports press shortcut like Meta+V to pop window on cursor.

But I think the horizontal layout of cosmic-utils/clipboard-manager is very good, so simply pop it on each workspace or workspace with active window is good for me.

**Comment by CarsonBurke** [comment]
I think an MVP would be to simply toggle show of all instances of the applet as normal on input of meta+v or similar. If this is acceptable I can try to PR @wiiznokes ?

**Comment by wiiznokes** [comment]
Sure, that would be a great mvp
